index.html

```diff
@@ -465,12 +481,20 @@
 					systemData.program = 
 						Math.round(data.system.sketchSpaceUsedBytes / 1024) + " kB / " + 
 						Math.round(data.system.sketchSpaceTotalBytes / 1024) + " kB (" + 
-						Math.round(100 * (data.system.sketchSpaceUsedBytes / data.system.sketchSpaceTotalBytes)) + "%)"; 
+						Math.round(100 * (data.system.sketchSpaceUsedBytes / data.system.sketchSpaceTotalBytes)) + "%)";
+					        if (Math.round(100 * (data.system.sketchSpaceUsedBytes / data.system.sketchSpaceTotalBytes)) >= 75)
```

Comment:
Can you just move this calc/check down to line 544, then you don't need a variable at all?